packages/navigator/src/browser/AbstractNavigatorTreeWidget.ts

```diff
@@ -0,0 +1,67 @@
+// *****************************************************************************
```

Comment:
The file should be renamed according to our [guidelines](https://github.com/eclipse-theia/theia/blob/master/doc/coding-guidelines.md#lower-case-names):

`abstract-navigator-tree-widget.ts`